openbb_platform/obbject_extensions/charting/openbb_charting/__init__.py

```diff
@@ -178,7 +178,7 @@ def to_chart(
         index = (
             data.index.name
             if has_data and isinstance(data, (pd.DataFrame, pd.Series))
-            else ""
```

Comment:
Do we need this? Empty string will also evaluate to `False`

Reply:
None is the valid setting for Index.